.github/uniswap-original/workflows/tests.yaml

```diff
@@ -35,7 +35,9 @@ jobs:
       - run: yarn build
         env:
           CI: false
-          REACT_APP_NETWORK_URL: 'https://mainnet.infura.io/v3/1221fd11e90849509afafd330ec7acc6'
+          REACT_APP_SUPPORTED_CHAIN_IDS="1,4"
+          REACT_APP_NETWORK_URL_1: 'https://mainnet.infura.io/v3/1221fd11e90849509afafd330ec7acc6'
+          REACT_APP_NETWORK_URL_4: 'https://rinkeby.infura.io/v3/1221fd11e90849509afafd330ec7acc6'
```

Comment:
2 questions:
1. Do you need to pdate the original CI files?
2. Why is xDai missing?

Reply:
> Do you need to pdate the original CI files?
nup, i don't. Good catch!


> Why is xDai missing?
We don't use for the test afaik